examples/cms-umbraco-heartcore/components/cover-image.js

```diff
@@ -0,0 +1,28 @@
+import cn from 'classnames'
+import Link from 'next/link'
+
+export default function CoverImage({ title, url, slug }) {
+  const image = (
+    <img
+      width={2000}
+      height={1000}
+      alt={`Cover Image for ${title}`}
+      className={cn('shadow-small', {
+        'hover:shadow-medium transition-shadow duration-200': slug,
+      })}
+      src={url}
+    />
+  )
+
+  return (
+    <div className="sm:mx-0">
+      {slug ? (
+        <Link as={slug} href="[...slug]">
```

Comment:
`as` isn't needed anymore.

```suggestion
        <Link href={slug}>
```

Reply:
updated